db/migrate/20141121164042_replace_arf_report_breakdown_view.rb

```diff
@@ -1,7 +1,9 @@
 class ReplaceArfReportBreakdownView < ActiveRecord::Migration
-   def self.up
-     execute <<-SQL
-CREATE OR REPLACE VIEW scaptimony_arf_report_breakdowns AS
+  def self.up
+    CreateScaptimonyArfReportBreakdowns.down
+    execute "DROP VIEW scaptimony_arf_report_breakdowns"
```

Comment:
i think i pushed to my branch at the same time you were merging.
i guess you dont have the latest version from my branch, but i don't think its an issue.